frontend/src/pages/ContactUsPage/components/Contacts/Contacts.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import { mockContacts } from '../../../../data/catalog';
+
+import styles from './Contacts.module.scss';
+
+const Contacts = ({ href }) => {
+  return (
+    <>
+      <div className={styles.contacts}>
+        <div className={styles['contacts-address']}>
```

Comment:
use dot notation for styles, as we do not use brackets anywhere in a project

Reply:
fixed